pkg/services/accesscontrol/resourcepermissions/store.go

```diff
@@ -670,6 +679,23 @@ func (s *store) createPermissions(sess *db.Session, roleID int64, resource, reso
 		p.Kind, p.Attribute, p.Identifier = p.SplitScope()
 		permissions = append(permissions, p)
 	}
+	/*
+		Add ACTION SET of managed permissions to in-memory store
+		// FIXME: make this only folder level of view, editor, admin
+	*/
+	if s.features.IsEnabled(context.TODO(), featuremgmt.FlagAccessActionSets) {
+		name, err := s.inMemoryActionSets.CreateActionSetName(resource, permission)
+		if err != nil {
+			return err
+		}
+		actionSet := s.inMemoryActionSets.GetActionSet(name)
```

Comment:
We only need the action set name, which you already get from calling `CreateActionSetName`, so this call to `GetActionSet` is not needed.